pulsar-broker/src/test/resources/log4j2.xml

```diff
@@ -19,18 +19,16 @@
 
 -->
 <configuration scan="true">
-<!-- 
```

Comment:
The change doesn't seem to be related.

Reply:
When I try to debug the API, using LoadBalancerTest to launch a multi nodes cluster and trying to change the debug level, such as org.apache.pulsar.broker.admin. but logback.xml configuration doesn't work and found that we actually using log4j2.xml which is configured in parent pom.

I just change the log config file by the way, it's ok if you still think we shouldn't change this

Reply:
revert